### PR TITLE
wxmac 3.1.4

### PR DESCRIPTION
**Pull request by karlvr**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/wxmac.rb [hunk 1]
```diff
@@ -1,25 +1,16 @@
...
   license "wxWindows"
-  revision 1
   head "https://github.com/wxWidgets/wxWidgets.git"
 
   livecheck do
     url :stable
     strategy :github_latest
   end
 
-  bottle do
```
Comment (SMillerDev) [inline Formula/wxmac.rb:15]: Please don't edit the bottle block

**Review by SMillerDev — CHANGES_REQUESTED** [review]

**Comment by karlvr** [comment]
Uh oh, those failures seem a little out of my league. This PR updates wxmac to 3.1.4... do I read those logs correctly that there are other packages that depend on version 3.0?

**Comment by SMillerDev** [comment]
It seems there are packages that depend on this formula. Can you add `revision 1` to all the formula that had a failing linkage check?

**Comment by carlocab** [comment]
See also #70125.

**Comment by karlvr** [comment]
@carlocab I'm sorry, I've done a poor job checking if there's already something happening for this package. I will close this PR.

**Comment by carlocab** [comment]
@karlvr, please split your second commit so that there is one commit per formula file modified. (See the [docs](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#create-your-pull-request-from-a-new-branch).) Thanks!

**Pull request closed without merging**